usecase/usecase-webapi-xwalk-tests/samples/IAPGooglePlay/index.html

```diff
@@ -44,23 +45,24 @@
   <h3 id="main_page_title"></h3>
 </div>
 <div class="content" style="margin: 20px 20px;">
-  Launch sub app "iap"
+  <p>Launch sub app "IAP Google Play Demo", choose "Google Play" and buy some in-app products.</p>
+  </br>
+  <h4>Pre-condition: </h4>
+  <ol>
+    <li>GMS (Google Mobile Services)</br>
```

Comment:
These precondition are duplicated with those documented in README.md file, suggest remove the README.md part.